src/main/java/com/ericsson/gerrit/plugins/eiffel/GerritModule.java

```diff
@@ -47,6 +47,9 @@ protected void configure() {
         bind(LifecycleListener.class).annotatedWith(UniqueAnnotations.create())
                                      .to(MessageQueueHandler.class);
 
+        // Register change listener that will send messages
+        DynamicSet.bind(binder(), EventListener.class).to(ChangeMergedEventListener.class);
```

Comment:
I must be missing something, I can't find any use of `PatchsetCreatedEventListener`

Reply:
Done in next commit.